core/src/main/java/com/scalar/db/api/TransactionCrudOperable.java

```diff
@@ -44,63 +46,129 @@ public interface TransactionCrudOperable {
 
   /**
    * Inserts/Updates an entry to the storage through a transaction with the specified {@link Put}
```

Comment:
Since I don't think `Insert` and `Update` are interchangeable, how about if we change this part as follows?:

```suggestion
   * Inserts an entry into or updates an entry in the underlying storage through a transaction with the specified {@link Put}
```